sdcm/cluster.py

```diff
@@ -2489,6 +2489,9 @@ def node_setup(self, node, verbose=False, timeout=3600):
         :param verbose:
         """
         node.wait_ssh_up(verbose=verbose)
+        if node.init_system == 'systemd' and (node.is_ubuntu() or node.is_debian()):
+            node.remoter.run('sudo systemctl disable apt-daily.timer')
```

Comment:
`systemctl disable` - prevents the service to run on next boot. Did you mean to disable and stop?

Reply:
https://askubuntu.com/questions/733469/what-is-the-difference-between-systemctl-start-and-systemctl-enable

Reply:
You are right, disable might be not enough. Fixed.